### PR TITLE
[ZH] Conclude OWC 2020

### PR DESCRIPTION
**Pull request by Kotoki1337**

### CODE REVIEW

**Comment by cl8n** [comment]
this note at the beginning can be removed
```markdown
*注意：比赛正在进行中，该翻译版本最后更新日期为 2020 年 11 月 30 日 17:30 UTC+08，最新赛程安排及图池请按[英文原文](/wiki/Tournaments/OWC/2020?locale=en)为准。*
```

**Comment by Kotoki1337** [comment]
resolved

File: wiki/Tournaments/OWC/2020/zh.md [hunk 1]
```diff
@@ -318,6 +302,22 @@ osu! 2020 世界杯由 osu! team 和多位社区成员举办。
 
 ## 比赛结果
 
+### 总决赛
+
+具体的半决赛统计信息可以在[谷歌试算表](https://docs.google.com/spreadsheets/d/1Csyk7zgrDF4iFJvgFRiBniCS6ZsQyXEkndNVUB65Un8/edit?rm=minimal "Google spreadsheet")中找到。
```
Comment (kj415j45) [inline wiki/Tournaments/OWC/2020/zh.md:307]:
```suggestion
具体的总决赛统计信息可以在[谷歌表格](https://docs.google.com/spreadsheets/d/1Csyk7zgrDF4iFJvgFRiBniCS6ZsQyXEkndNVUB65Un8/edit?rm=minimal "谷歌表格")中找到。
```
`Google spreadsheet` -> `谷歌表格` 或 `Google 表格`，下同。

File: wiki/Tournaments/OWC/2020/zh.md [hunk 2]
```diff
@@ -318,6 +302,22 @@ osu! 2020 世界杯由 osu! team 和多位社区成员举办。
...
+
+2020 年 12 月 5 日，星期六：
+
```
Comment (kj415j45) [inline wiki/Tournaments/OWC/2020/zh.md:310]: blank line needed?

**Review by TPGPL — APPROVED** [review]

**Pull request merged**